src/ert/gui/simulation/ensemble_experiment_panel.py

```diff
@@ -85,18 +85,27 @@ def __init__(
 
         design_matrix = analysis_config.design_matrix
         if design_matrix is not None:
+            use_dm_checkbox = QCheckBox("Use Design Matrix")
+            use_dm_checkbox.setObjectName("use-dm-checkbox")
+
             show_dm_param_button = QPushButton("Show parameters")
             show_dm_param_button.setObjectName("show-dm-parameters")
             show_dm_param_button.setMinimumWidth(50)
 
             button_layout = QHBoxLayout()
+            button_layout.addWidget(use_dm_checkbox)
             button_layout.addWidget(show_dm_param_button)
             button_layout.addStretch()  # Add stretch to push the button to the left
 
             layout.addRow("Design Matrix", button_layout)
             show_dm_param_button.clicked.connect(
                 lambda: self.on_show_dm_params_clicked(design_matrix)
             )
+            use_dm_checkbox.stateChanged.connect(
+                lambda: self.on_use_dm_checkbox_toggled(
+                    use_dm_checkbox.checkState(), design_matrix
```

Comment:
I wonder if we have to do `use_dm_checkbox.checkState()` or if the new state is already given through the emitted signal, so we can include it as an argument to the lambda. Can we double check this?

Reply:
I'm going to remove the checkbox as it is not really necessary I think.